Qt6 port, warning suppression

kunzjacq opened the pull request:
In addition to changes required to port to Qt6, warnings related to
boost bind and to deprecated types in Tiff{Reader,Writer}.hpp were
eliminated.

mirusu400 committed:
Add Korean Translate

maltaisn committed:
Add French translation

fyrtax committed:
Add Polish translation

Commit:
Ignore build folder

Commit:
German translation added

Commit:
Merge remote-tracking branch 'polish/master'

Commit:
Merge remote-tracking branch 'french/master'

Commit:
Add option to use black as filling background color

Commit:
Better translation

Commit:
Add 1200 DPI as option

Commit:
Fix for #87af484 which broke the build

Commit:
Merge remote-tracking branch 'korean/master'

Commit:
Update readme

Commit:
German translation synced with project and translated missing phrases

Commit:
Translations synced with project

Commit:
Bump version

kunzjacq committed:
Qt6 port, warning suppression

In addition to changes required to port to Qt6, warnings related to
boost bind and to deprecated types in Tiff{Reader,Writer}.hpp were
eliminated.

leonleon33156 commented:
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

mara004 commented:
Wow, this looks very interesting. Thanks for the work.
Unfortunately, this repository appears to be abandoned (see #170). You could consider also submitting the Pull Request to https://github.com/vigri/scantailor-advanced which might become some kind of community-maintained fork.

kunzjacq committed:
Enable compilation with both Qt5 and Qt6

Tested with Qt 5.15.2 and 6.2.3.

kunzjacq committed:
Require at least Qt 5.15

kunzjacq committed:
Enable compilation with Qt5 starting from version 5.9

kunzjacq committed:
misc fix regarding the use of a deprecated function

kunzjacq committed:
misc fix regarding the use of a deprecated function (take two)

vigri committed:
Update README.md

Links updated

kunzjacq committed:
Enable compilation with Qt6 and MSVC

Add missing headers, account for the different target path, change the
Qt binary dir determination method.

kunzjacq committed:
fix install with MinGW GCC from command-line

kunzjacq committed:
ignore CMake user settings

kunzjacq committed:
Enable the installation of scantailor-advanced with libjpeg-turbo under
Windows

Enable the CMake install step to find the libjpeg-turbo dll, which has
version number (libjpeg-62.dll)

kunzjacq committed:
Merge branch 'master' into Qt6

kunzjacq committed:
Add missing include for Qt 6.3.0 compatibility

a17r commented:
What's the status here? Such a port would be much appreciated while distributions are already in process of cleaning up Qt5 revdeps.
